test-case/manual/DPMST.md

```diff
@@ -0,0 +1,70 @@
+# DPMST test
+DisplayPort Multi-Stream Transport is a manual test.
+
+# Preconditions
+1. Hardware requirements met for each DPMST case
+
+# Test Description
+* Test output from one test unit to (minimum of) two separate monitors with audio
+  output via DP (as well as with USB-C and HDMI adapters if monitors have those
+  inputs available).
+
+# 1) Single DP -> Multi DP hub Test Case:
+## Hardware Requirements
+1. One DP-MST hub with single DP input -> multi-DP output
+2. Two monitors with DP input and audio output
+3. Two DP-USB-C adapters (if monitors also have USB-C input)
+4. Two DP-HDMI adapters (if monitors also have HDMI input)
+* If DP-MST hub has additional outputs, test can be extended to include
+  additional monitors as availble
+
+## Run Instructions
+1. Two monitors are connected via one DP hub using DP-DP cables.
+2. Run:
+   ```
+   amixer contents | grep -i jack -A 2
+   ```
+   to ensure DP jacks on DP hub are detected.
+3. Run aplay, using the 2 DP jacks found with `amixer contents` command.
+   ```
+   aplay -D $dev(jack1) -r $rate -c 2 -f $fmt -d 3 resource/raw/test<specified>.raw
+   aplay -D $dev(jack2) -r $rate -c 2 -f $fmt -d 3 resource/raw/test<specified>.raw
+   ```
+4. Connected monitors should both show display and play audio.
+5. Check dmesg for any unexpected errors.
+* If monitors have USB-C / HDMI input as well, test can also be run with:
+  * DP hub -> DP cables + DP-USB-C adapters -> monitors
+  * DP hub -> DP cables + DP-HDMI adapters -> monitors
+
+# 2) Daisy Chain Test Case:
+## Hardware Requirements
+1. One monitor with DP input **AND** ouput, as well as audio output (monitor1)
```

Comment:
It is better to mention to enable DP1.2 in the monitor OSD menu. Many cases failure of DP MST chain test is because of the wrong setting in monitor.

Reply:
I'll add that comment. Thanks!

Reply:
added